corona-warn-app/cwa-app-android
# Camera permission handling (EXPOSUREAPP-11227)

This PR is related to EXPOSUREAPP-11227.
Content:
- Fixes an issue when a permanent deny handling dialog appeared without any need (after first app start).
- Cleans obsolet Camera Permssion Provider (led to weired bugs in edge case situations)

How to test:
- Fresh install of the App
- Open the UniversalQRCodeScanner
- Check permission dialog and handling
